faunadb/objects.py

```diff
@@ -170,6 +171,16 @@ class Query(_Expr):
   See the `docs <https://app.fauna.com/documentation/reference/queryapi#special-type>`__.
   """
 
+  def __init__(self, value):
+    """
+    :param value:
+      If an api_version key is not defined, `faunadb.__api_version__` is used.
+    """
+    if isinstance(value, dict):
+      if not "api_version" in value:
+        value["api_version"] =  __api_version__
```

Comment:
please don't do this, we should not use the default api version (`v3`) case it was not provided, ideally, users shouldn't be constructing query types